.github/workflows/unit-tests.yml

```diff
@@ -176,6 +176,7 @@ jobs:
           rdk/ds/frontPanelIndicator.hpp
           rdk/ds/frontPanelConfig.hpp
           rdk/ds/frontPanelTextDisplay.hpp
+		  rdk/ds/motionDetection.hpp
```

Comment:
I could not find any use of motionDetection.hpp in the project
please remove it